src/pharmpy/model/distributions/symbolic.py

```diff
@@ -316,14 +316,17 @@ def __init__(
 
     @classmethod
     def create(cls, names: Sequence[str], level: str, mean, variance):
+        variance = Matrix(variance)
+
+        if variance.is_symbolic_covariance_matrix() is not True:
```

Comment:
We could put this into the `is_positive_semidefinite` method instead. If not symmetric return `False` and if symbolic return `None`. `symengine` has a method `is_positive_definite` that should be fast, but I don't think we can use it in this case.